alert/store.go

```diff
@@ -255,9 +260,63 @@ func NewDB(ctx context.Context, db *sql.DB, logDB alertlog.Store) (*DB, error) {
 			FROM services
 			WHERE id = $1
 		`),
+
+		findPendingNotifications: p(`
+			SELECT
+			COALESCE(ucm.type::text, nc.type::text) AS dest_type,
```

Comment:
These should be scanned separately, something like email would be in both in the future.

`engine/message/db.go:401` for an example

Reply:
Not exactly, I was only referring to the columns on that line -- separate them in the select instead of using `coalesce` for the type info